cmd/agent/main.go

```diff
@@ -52,6 +56,16 @@ func run(cmd *cobra.Command, args []string) error {
 		snapshotURL = fmt.Sprintf(DefaultSnapshotURLFmt, entrypoint.ExternalSnapshotPort)
 	}
 
+	diagnosticsURL := os.Getenv("AES_DIAGNOSTICS_URL")
+	if diagnosticsURL == "" {
+		diagnosticsURL = fmt.Sprintf(DefaultDiagnosticsURLFmt, AdminDiagnosticsPort)
+	}
+
+	reportDiagnostics := os.Getenv("AES_REPORT_DIAGNOSTICS_TO_CLOUD")
```

Comment:
I think the default should be to not report diagnostics to the cloud due to concerns from the types of customers I mentioned in my previous message. Then the `helm upgrade` commands in the cloud that have you setup a cloud connect token can set the environment variable to `opt-in`. That said if there is a more compelling reason to have the default be `opt out` I don't think it will be too much of a problem so long as these changes are included in the Edge-Stack release notes so that any customers who care can preemptively set the variable to `opt-out` when they upgrade.

Reply:
`reportDiagnostics := os.Getenv("AES_REPORT_DIAGNOSTICS_TO_CLOUD")` should also probably take whatever default value we want to go with whenever `AES_REPORT_DIAGNOSTICS_TO_CLOUD` is not set so we don't mess up the customers using YAML installs.

Reply:
Will add defaults in the follow-up PR once we decide which route to take